stdlib/FileWatching/test/runtests.jl

```diff
@@ -176,24 +163,24 @@ file = joinpath(dir, "afile.txt")
 # initialize a watch_folder instance and create afile.txt
 function test_init_afile()
     @test isempty(FileWatching.watched_folders)
-    @test @elapsed(@test(watch_folder(dir, 0) == ("" => FileWatching.FileEvent()))) <= 2
+    @test(watch_folder(dir, 0) == ("" => FileWatching.FileEvent()))
     @test @elapsed(@test(watch_folder(dir, 0) == ("" => FileWatching.FileEvent()))) <= 0.5
     @test length(FileWatching.watched_folders) == 1
     @test unwatch_folder(dir) === nothing
     @test isempty(FileWatching.watched_folders)
-    @test 0.001 <= @elapsed(@test(watch_folder(dir, 0.004) == ("" => FileWatching.FileEvent()))) <= 2
-    @test 0.001 <= @elapsed(@test(watch_folder(dir, 0.004) == ("" => FileWatching.FileEvent()))) <= 0.5
+    @test 0.002 <= @elapsed(@test(watch_folder(dir, 0.004) == ("" => FileWatching.FileEvent())))
+    @test 0.002 <= @elapsed(@test(watch_folder(dir, 0.004) == ("" => FileWatching.FileEvent()))) <= 0.5
```

Comment:
Should this `0.5` be removed as well?

Reply:
No, `0.004` is less than our internal threshold of `0.01`, so this is actually just checking `@elapsed sleep(0.004)` and not testing the filesystem performance

Reply:
So, should we remove this test then, if it's not testing filesystem performance?